src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html

```diff
@@ -0,0 +1,505 @@
+<!--TODO: Fix links-->
+<div>
+  <h2 class="color-blue" *ngIf="showQuestion.length === 0 || showQuestion.includes(true)">Questions</h2>
+  <!-- Question -->
+  <div class="instr-help-qn0" *ngIf="showQuestion.length === 0 || showQuestion[0]" #question>
```

Comment:
You can make use of the work done in #9395: use `displaySubsection` instead.

Reply:
Done!